app/labor/markdown_fixer.rb

```diff
@@ -36,6 +36,12 @@ def modify_hr_tags(markdown)
       end
     end
 
+    def lowercase_published(markdown)
+      markdown.gsub(/-{3}.*?-{3}/m) do |front_matter|
+        front_matter.gsub(/^published: /i, "published: ")
```

Comment:
This is:
`/^published: /i`
`/newline,published,colon,1-whitespace-character/case-insensitive-flag`